src/DrawManager.sol

```diff
@@ -98,8 +98,8 @@ contract DrawManager {
   /// @notice The maximum total rewards for both auctions for a single draw
   uint256 public immutable maxRewards;
 
-  /// @notice The address to allocate any remaining reserve from each draw.
-  address public immutable remainingRewardsRecipient;
+  /// @notice The address of the staking vault to contribute remaining reserve on behalf of
+  address public immutable stakingVault;
```

Comment:
I think this should be renamed to just "vault"; because the staking vault is a very specific thing: pt-v5-staking-vault

Reply:
Renamed to "vaultBeneficiary": [b5048de](https://github.com/GenerationSoftware/pt-v5-draw-manager/pull/9/commits/b5048de2e03f0970677d2e0fe85812f89c356e1d)